go.mod

```
@@ -0,0 +1,22 @@
+module github.com/cf-lookup-route
```

The module should be:

```suggestion
module github.com/cloudfoundry/cf-lookup-route
```

---

this was marked resolved but not changed. The comment is relevant though.

---

Sorry, have fixed that in the other branch already